android/tangram/build.gradle

```diff
@@ -14,44 +7,51 @@ version = VERSION_NAME
 apply from: 'versioning.gradle'
 
 android {
-  compileSdkVersion 24
-
-  /*
-   * Ubuntu can't run the aapt on 64 bit before installing this packages
-   *    sudo apt-get install lib32stdc++6
-   *    sudo apt-get install lib32z1
-   */
-
-  buildToolsVersion '24.0.2'
+  compileSdkVersion 25
+  buildToolsVersion '25.0.0'
 
   defaultConfig {
     minSdkVersion 15
-    targetSdkVersion 24
+    targetSdkVersion 25
     versionCode buildVersionCode()
     versionName VERSION_NAME
+    externalNativeBuild {
+      cmake {
+        targets 'tangram'
+        arguments '-DPLATFORM_TARGET=android',
+                  '-DANDROID_TOOLCHAIN=clang',
+                  '-DANDROID_STL=c++_shared'
+        cppFlags '-std=c++14',
+                 '-pedantic',
+                 '-fPIC',
+                 '-fexceptions',
+                 '-frtti'
+        abiFilters 'armeabi-v7a'
+      }
+    }
   }
 
-  sourceSets.main {
-    manifest.srcFile 'AndroidManifest.xml'
-    java.srcDirs = ['src']
-    jni.srcDirs = []
-    jniLibs.srcDirs = ['libs']
+  externalNativeBuild {
+    cmake {
+      path '../../CMakeLists.txt'
+    }
   }
-}
 
-// Add gdb server to apk
-afterEvaluate {
-    Sync packageTask = project.getTasks().findByName("packageReleaseJniLibs")
```

Comment:
I think the file we had can go away with it too, `toolchains/android.gdb.cmake`.